src/App.tsx

```diff
@@ -0,0 +1,11 @@
+import { Form } from "../lib/main";
+
+const App = (): JSX.Element => {
+  return (
+    <>
+      <Form />
+    </>
+  );
+};
+
+export default App;
```

Comment:
Not sure what your opinion is on them but personally I'm not a fan of default exports

Reply:
I agree. I am in the habit of doing a default export for App but I don't have a good reason for it really. Happy to make it a regular export